pkg/modelregistry/jsonvalues/convertjson.go

```diff
@@ -109,9 +109,9 @@ func CorrectJSONPaths(jsonBase string, jsonPathValues []*change.ConfigValue,
 				break
 
 			} else if hasPrefixMultipleIdx(jsonPathWildIndex, modelPathWildIndex) {
-				jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
+				//jsonSubPath := jsonPathWildIndex[len(modelPathWildIndex):]
 
-				newTypeValue, err := assignModelType(paths, modelPath, jsonSubPath, jsonPathValue)
+				newTypeValue, err := assignModelType(paths, modelPath, jsonPathValue)
```

Comment:
I see no difference in the usage of the method from before but the following lines are different, is it worth consolidating by extracting this line and the error evaluation or am I missing some logic.

Reply:
updated